testnet/testnet-deploy-contracts.sh

```diff
@@ -52,20 +52,30 @@ fi
 # deploy contracts to the geth network
 echo "Deploying contracts to the geth network..."
 docker network create --driver bridge node_network || true
+
+# deploy Obscuro management contract
 docker run --name=contractdeployer \
     --network=node_network \
     --entrypoint /home/go-obscuro/tools/contractdeployer/main/main \
      testnetobscuronet.azurecr.io/obscuronet/obscuro_contractdeployer:latest \
-    --l1NodeHost=${l1host} \
-    --l1NodePort=${l1port} \
+    --nodeHost=${l1host} \
+    --nodePort=${l1port} \
+    --contract=MGMT \
```

Comment:
Nit: Can this be wrapped in double quotes "MGMT" ? Make is clear that it's a hardcoded value.

Reply:
Cool, yeah will give it a go. If those quotes get passed into the flag args string then I'm not so sure though, don't want to have `"\"MGMT\""` as the Go constant ideally haha